### PR TITLE
Fixed problems in ISISIndirectBayesTest.QLWidthTest system test

### PR DESCRIPTION
**Pull request by SilkeSchomann**
Fix for [#30062](https://github.com/mantidproject/mantid/issues/30062) 

Changed file names to avoid interference caused by using the same file name for three different tests. Also ensured that the _cleanup_files routine tries to delete the correct files.

To Test: Run system tests ISISIndirectBayesTest to check whether the call for  _cleanup_files succeeds. In addition, monitor results of Windows nightly builds to verify whether it still randomly fails for ISISIndirectBayesTest.

This does not require release notes because the change only impacts the system test.
---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Testing/SystemTests/tests/framework/ISISIndirectBayesTest.py
Comment (Harrietbrown) [inline Testing/SystemTests/tests/framework/ISISIndirectBayesTest.py:230]: Does the Parameters file no longer need removing?

Comment (SilkeSchomann) [reply Testing/SystemTests/tests/framework/ISISIndirectBayesTest.py:230]: There are no Parameters file left once the test finished. So clean up failed for it every time.

**Review by Harrietbrown — APPROVED** [review]
The tests pass on my system, All that remains is to moniter the test once merged to ensure it is a consistent fix. Good Job

**Review by DavidFair — APPROVED** [review]
Thanks, fixes like this really do help the developer experience without people realising, since we should have to sit waiting for CI less
Lets :shipit:

**Pull request merged**
